scripts/generateDoc.sh

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at:
+#
+#     https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
```

Comment:
Please use trap or set -e to exit the script if an error occurs (see rest of scripts in the repo to provide a consistent approch)